deploy/backup-s3.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cluster1-s3-credentials
+type: Opaque
+stringData:
```

Comment:
In PSMDB and PXC we use `data` type here (https://github.com/percona/percona-server-mongodb-operator/blob/main/deploy/backup-s3.yaml#L6), should we change it here also to be consistent? I was pasting base64 encoded values and didn't notice they were encoded again 😂

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/77/commits/d4b24e1eef1f8b14e61161247256d7c9b660bc2d